mantidproject/mantid
# Initial Pearl Powder Diffraction Clean Up

**Description of work.**
- [x] Merged #16614
- [x] Re-run system test build

This PR cleans up the legacy Pearl Powder Diffraction script to an extend, the script contains repetition / loops of algorithms calls which were not required and been resolved. `Focus v1` and `Focus v2` now utilises the same function.

**To test:**
Check that all system test pass
Code changes look acceptable
Run some of the scripts provided by Craig locally
Check if any further refactoring that could be made here



Fixes #16460 
_Does not need to be in the release notes._



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
